x/gamm/types/msgs_test.go

```diff
@@ -867,3 +872,148 @@ func TestMsgExitSwapShareAmountIn(t *testing.T) {
 		}
 	}
 }
+
+// Test authz
+func TestAuthzMsg(t *testing.T) {
+	appParams.SetAddressPrefixes()
+	pk1 := ed25519.GenPrivKey().PubKey()
+	addr1 := sdk.AccAddress(pk1.Address()).String()
+	coin := sdk.NewCoin("stake", sdk.NewInt(1))
+	someDate := time.Date(1, 1, 1, 1, 1, 1, 1, time.UTC)
+
+	testCases := []struct {
```

Comment:
can we have names for the test cases please?

Reply:
I actually don't think names mattered here, its pretty clear & automatically derivable from the message type, but not really an issue

Reply:
On another lookg, agreed